src/model/import.ts

```diff
@@ -59,6 +59,9 @@ export class ScannedImport implements Resolvable {
   }
 
   resolve(document: Document): Import|undefined {
+    if (!document.analyzer.canResolveUrl(this.url)) {
```

Comment:
To answer your question though, I don't think the current approach causes downstream problems, given this is the existing behavior for URLs that can't be loaded.